dmscripts/publish_draft_services.py

```diff
@@ -111,13 +111,11 @@ def copy_draft_documents(
         client.update_service(service_id, document_updates, user='publish_draft_services.py')
 
 
-def _publish_draft_service(
+def publish_draft_service(
     client,
     draft_service,
-    framework_slug,
-    copy_documents_callable=None,
+    _assets_endpoint,
     dry_run=True,
-    skip_docs_if_published=True,
```

Comment:
Minor issue, but this change probably belongs in the previous commit

Reply:
Yeah I haven't quite got my step by step commit splitting down but I'm getting there :)